docs/zkmeta-official.md

```diff
@@ -0,0 +1,122 @@
+# ZK-Meta and X-Matrix Labs
+
+Authors: Chris Guo, Kyrie Lu, Nathan Yang
+
+## Table of Contents
+* [What is ZK-Meta](#what-is-zk-meta)
+* [What goals is ZK-Meta trying to accomplish](#what-goals-is-zk-meta-trying-to-accomplish)
+* [What makes ZK-Meta so special](#what-makes-zk-meta-so-special)
+* [ZK-Meta architecture](#zk-meta-architecture)
+* [What is X-Matrix Labs](#what-is-x-matrix-labs)
+* [What are X-Matrix Labs's main focuses](#what-are-x-matrix-labss-focuses)
+    * [Gateway](#gateway)
+    * [Privacy](#privacy)
+    * [De-Fi](#de-fi)
+    * [Storage](#storage)
+* [References](#references)
+    * [Privacy related](#privacy-related)
+    * [Labs design](#labs-design)
+
+## What is ZK-Meta?
+ZK-Meta is a privacy protection computing framework developed by X-Matrix Labs.
+In essence, ZK-Meta is a Web3.0 privacy protection infrastructure that aims to protect user and business data
+in lieu of the General Data Protection Regulation (GDPR).
+ZK-Meta provides customized privacy preserving solutions suitable for various businesses.
+In addition, ZK-Meta offers pre-optimized technical selections and solutions.
+Currently, ZK-Meta includes Zero-Knowledge Proof (ZKP) protocols such as ZK-3D and ZK-Rollup.
```

Comment:
The name of Zk-3D compoment we should disscuss.  It's not only include add/sub , mul/div , big/small，and range proof ..

Reply:
Not 100% sure what you mean here...I thought the 4 arithmetic operations, size comparison, and range proof **would** be implemented in the future 🤔